README.md

```diff
@@ -1,7 +1,7 @@
 <h1>
   <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="docs/images/nf-core-riboseq_logo_dark.png">
-    <img alt="nf-core/riboseq" src="docs/images/nf-core-riboseq_logo_light.png">
+    <source media="(prefers-color-scheme: dark)" srcset="docs/images/nf-core-riboseq_logo_light.png">
+    <img alt="nf-core/riboseq" src="docs/images/nf-core-riboseq_logo_dark.png">
```

Comment:
this should not have changed

Reply:
ah right, so the naming was not dark because of dark font, but dark meant white because of dark theme. Fair enough. Do I have to switch it back?

Reply:
sorted